jobs/batch_operation_sample.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python
+
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START instantiate]
+from googleapiclient.discovery import build
+
+client_service = build('jobs', 'v2')
+
+# [END instantiate]
+
+
+# [START batch_job_create]
+def batch_job_create(client_service, company_name):
+    import base_job_sample
```

Comment:
I understand your concern here; but for clarity I do feel that this part can be replaced with two simple `job` variables so that developers do not need to read another file to understand what's going on behind the scenes.

Reply:
It could be changed later since it request modifying the samples in all languages.